src/lib/media/VoiceO.ts

```diff
@@ -0,0 +1,625 @@
+import { CallDirection } from "$lib/enums"
```

Comment:
do we need this file? src/lib/media/VoiceO.ts or was this for testing?


if so, can we update the naming?